yarn-project/circuits.js/src/structs/kernel/combined_accumulated_data.ts

```diff
@@ -98,16 +98,16 @@ export class CombinedAccumulatedData {
     return CombinedAccumulatedData.fromBuffer(Buffer.from(str, 'hex'));
   }
 
-  static empty() {
+  static default() {
     return new CombinedAccumulatedData(
-      makeTuple(MAX_NEW_NOTE_HASHES_PER_TX, SideEffect.empty),
-      makeTuple(MAX_NEW_NULLIFIERS_PER_TX, SideEffectLinkedToNoteHash.empty),
-      makeTuple(MAX_NEW_L2_TO_L1_MSGS_PER_TX, Fr.zero),
-      Fr.zero(),
```

Comment:
The inconsistency here hurts me =/ I get the `Fr.default()` and `Fr.ZERO` for comparisons, but mixing them up on the same struct...